benchmarks/kraken/sunspider-compare-results

```diff
@@ -55,7 +55,7 @@ GetOptions('shell=s' => \$jsShellPath,
 $suite = "ubench" if ($ubench);
 $suite = "v8-v4" if ($v8suite);
 $suite = "parse-only" if ($parseOnly);
-$suite = "sunspider-0.9.1" if (!$suite);
+$suite = "kraken-1.1" if (!$suite);
```

Comment:
at line 41 there is:
--suite           Select a specific benchmark suite. The default is sunspider-0.9.1

Can you also update that?